tutorials/hardhat-tutorial.md

```diff
@@ -0,0 +1,425 @@
+
+### Configuring the Test File {: #configuring-the-test-file }
+
+To get started, create a folder called `tests` and a file named `Dao.js`. Then copy and paste the contents below to set up the initial structure of your test file. Be sure to read the comments as they can clarify the purpose of each line. 
+
+``` javascript
+const { ethers } = require("hardhat");
+require("@nomicfoundation/hardhat-toolbox");
+
+// Import Chai to use its assertion functions here.
+const { expect } = require("chai");
+
+// Use `loadFixture` to share common setups (or fixtures) between tests.
+const { loadFixture } = require("@nomicfoundation/hardhat-network-helpers");
+
+// Indicate the collator the DAO wants to delegate to
+const targetCollator = "{{ networks.moonbase.staking.candidates.address1 }}";
+```
+
+### Using Fixtures {: #using-fixtures }
+
+A fixture is a function that configures the blockchain to a desired state. For example, our initial tests for the StakingDAO will be run after the StakingDAO has been deployed but before any contributions have been made. When a fixture is loaded for the first time, it is executed as normal to properly configure the state of the chain. When called a second time, the fixture is not executed again but rather the state of the network is reset to the point in time immediately after the fixture was initially executed. In other words, this resets state changes prior to the initial execution of the fixture and saves time.
+
+The first fixture in the test file represents an "empty" stakingDAO but the second fixture will have one member (besides the DAO admin). Fixtures allow you to quickly switch between different states, simplifying and speeding up the testing process. In this tutorial only two fixtures are specified but it's easy to imagine other states you'd like to test, such as a scenario with 100 DAO members or one with multiple admins of the DAO. 
+
+To define the two fixtures, add the following snippet to your test file:
+
+``` javascript
+// `describe` is a Mocha function that allows you to organize your tests. All Mocha
+// functions are available in the global scope.
+//
+// `describe` receives the name of a section of your test suite, and a
+// callback. The callback must define the tests of that section. This callback
+// can't be an async function.
```

Comment:
I would put the describe info outside of the comments so we can link back to Mocha's docs, but not sure it really matters here 🤷‍♀️